### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by K-mone**
In this branch, I have completed the following tasks:

- Created the .gitignore file and added node modules to this file
- Set up the HTML and CSS linters
- Updated the nav bar, header, project sections and contact section
- Added the style for above section
- Added the required photos

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    </div>
+    <div class="hero">
+      <p>Hello, I am</p>
+      <h1>Kyae Mone Win</h1>
+      <h2 class="mb-5">Front-end developer based in Myanmar</h2>
+
+      <div>
+        <a href="#" class="btn btn-branding">Contact me</a>
+      </div>
+    </div>
```
Comment (K-mone) [inline index.html]: I think it would be better to use header tag instead of using div element

File: index.html [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+    <p class="text-center m-5">
+      Featured Project which my best projects be choice to show
+    </p>
+    <div class="container">
```
Comment (K-mone) [inline index.html:79]: Using section element would look better your code

File: index.html [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+            ><i class="fa-brands fa-instagram"></i
+          ></a>
+          <a
+            href="https://www.linkedin.com/in/kyayemone-win-2bb151217"
+            target="_blank"
+            title="LinkedIn profile"
+            rel="noopener"
+            ><i class="fa-brands fa-linkedin"></i
+          ></a>
+        </div>
```
Comment (K-mone) [inline index.html]: I think it would be better as listed items

File: index.html [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+        <p class="text-center mt-5">
+          This project was coded by K and it is
+          <a
+            href="https://github.com/K-mone/Figma-Portfoilo"
+            target="_blank"
+            title="Github source code"
+            rel="noopener"
+            >open-sourced</a
+          >
+        </p>
```
Comment (K-mone) [inline index.html]: In my opinion, small element sounds better than p element

File: style.css [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+body {
+  font-family: var(--default-font-family);
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
```
Comment (K-mone) [inline style.css]: Please remove the unused tags

File: style.css [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+.btn-branding-outline {
+  color: var(--branding-color);
+  border: 1px solid var(--branding-color);
+  border-radius: 4px;
+  font-size: 18px;
+  line-height: 27px;
+  padding: 15px 30px;
+}
+
```
Comment (K-mone) [inline style.css:158]: On my point of view, you should also add :hover property for above class to look better when you hover it

**Review by HtetWaiYan7191 — APPROVED** [review]
Hi @K-mone  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 Highlights
-  Descriptive PR ✔️
- Well Structured Code ✔️
- Correct Git flow ✔️

## Optional suggestions
[Optional] While reviewing, always provide the reasons why you want to make changes. This can help the person requesting for the review understand why these changes are needed 🚀 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**